### ISSUE TITLE
[Help]: keep crashing without any error info

### ISSUE THREAD

**Issue by JerryPong0410** [0x90d/videoduplicatefinder#481]
## Environment
 - OS: [e.g. Windows / Linux / MacOS ]
windows10

## Question
A clear and concise detailed question.
i use"native ffmpeg"binding", at first it can finish scan. Then i scan more files more folder, it starts to crash with no error after the scan goes like more than 10 mins. But at the bottom, it still show how the scan goes (remaing time ...) but doesn't ↓↓↓↓↓↓↓↓↓

Then i try to redownload, now it still crashes even though i scan a few files which is not as usual as the begining. When i start scanning, after the screen "please wait" for 2-3 sec, it crashes to desktop without any info.  also without any details of how the scan goes

im really sorry that im not native speaker, also i dont understand  what some words mean on PC or program languages... anything like that.

When i finished downloading, i know it won't work if there is no folder "bin doc include presets" in vdf folder so i have to download those file especially with the name "share version" included. Same happens.

i try lot of files replacing again and again, keep crashing. now i dont know what to do. Also i wanna know what these file are and for what : when should i use them
![image](https://github.com/0x90d/videoduplicatefinder/assets/52005549/aba5dedb-a5b9-42ea-bf61-91bf3a2db47d)
[

what are these and what are the contrast of downloading here and the biggese green" download"
![屏幕截图 2024-01-14 125633](https://github.com/0x90d/videoduplicatefinder/assets/52005549/fcd2e81f-a15b-4552-be2e-d7f16339ada1)
](url)
![屏幕截图 2024-01-14 125837](https://github.com/0x90d/videoduplicatefinder/assets/52005549/19296866-c8fa-4124-b9bb-dbb4d056d345)

## Snapshots
Optionally, include snapshots to help clarify the question.

**Comment by 0x90d** [comment]
> Same happens.

Did you also unchecked "Use native ffmpeg binding" ? 
If you say it crashes does the VDF window disappears? In this case please post the .NET crash details from windows event viewer.

**Comment by JerryPong0410** [comment]
No i need to use native ffmpeg coz It's way more faster. Yup vdf Windows disappears.

It ran perfectly until one day it said something that i kinda forgot. I know that required me to download latest ffmpeg. So i did. Then things started to go wrong.

Then i redownload a whole software.

According to the software appearance, it seem like it updated?? But the date on release page is still Aug.17 2021 

https://github.com/0x90d/videoduplicatefinder/assets/52005549/46d81304-1ca7-4882-bf22-d209c0381ebc

**Comment by JerryPong0410** [comment]
Hello, i unchecked "native fmpeg" last night. it said it'd take 10 hrs which is 8 hrs more than i checked. Then i went to sleep.  Just now I got up and found that still crashed with no error info.

**Comment by 0x90d** [comment]
Native ffmpeg binding can easily crash on corrupted video files. There is already an opened issue about this. Please try and split up the scans so each scan only scans some of your files. If a scan completes without a crash these files are fine. If it crashes again then split up these files again until you found which file causes it.

**Comment by JerryPong0410** [comment]
> Native ffmpeg binding can easily crash on corrupted video files. There is already an opened issue about this. Please try and split up the scans so each scan only scans some of your files. If a scan completes without a crash these files are fine. If it crashes again then split up these files again until you found which file causes it.

Is it still on developing? Really expect a new version

**Comment by vxfranky** [comment]
Same issue. Is it possible to check file integrity before video scan?

**Comment by 0x90d** [comment]
@vxfranky Yes it is. Don't use native ffmpeg binding and then check the log file for files where ffmpeg has failed.
